pysite/views/error_handlers/http_5xx.py

```diff
@@ -13,28 +13,17 @@ class Error500View(ErrorView):
     def get(self, error: HTTPException):
         error_desc = ERROR_DESCRIPTIONS.get(error.code, "We're not really sure what happened there, please try again.")
 
-        return render_template("errors/error.html", code=error.code, req=request, error_title=error_desc,
-                               error_message="An error occurred while "
-                                             "processing this "
-                                             "request, please try "
-                                             "again later. "
-                                             "If you believe we have made a mistake, "
-                                             "please "
-                                             "<a href='https://github.com"
-                                             "/discord-python/site/issues'>file an issue on our GitHub"
-                                             "</a>."), error.code
+        return self.render("errors/error.html", code=error.code, req=request, error_title=error_desc,
+                           error_message="An error occurred while processing this request, please try "
+                                         "again later. If you believe we have made a mistake, please "
+                                         "<a href='https://github.com/discord-python/site/issues'>file an issue on our"
+                                         " GitHub</a>."), error.code
 
     def post(self, error: HTTPException):
-        error_desc = ERROR_DESCRIPTIONS.get(error.code,
-                                            "We're not really sure what happened there, please try again.")
+        error_desc = ERROR_DESCRIPTIONS.get(error.code, "We're not really sure what happened there, please try again.")
 
-        return render_template("errors/error.html", code=error.code, req=request, error_title=error_desc,
-                               error_message="An error occurred while "
-                                             "processing this "
-                                             "request, please try "
-                                             "again later. "
-                                             "If you believe we have made a mistake, "
-                                             "please "
-                                             "<a href='https://github.com"
-                                             "/discord-python/site/issues'>file an issue on our GitHub"
-                                             "</a>."), error.code
+        return self.render("errors/error.html", code=error.code, req=request, error_title=error_desc,
+                           error_message="An error occurred while processing this request, please try "
+                                         "again later. If you believe we have made a mistake, please "
+                                         "<a href='https://github.com/discord-python/site/issues'>file an issue on our"
```

Comment:
These lines get *very* close to 120 characters. Maybe shove everything after the `>` to the line below?